src/main/kotlin/no/nav/familie/ef/sak/ekstern/stønadsperiode/EksternStønadsperioderService.kt

```diff
@@ -45,9 +46,56 @@ class EksternStønadsperioderService(
 
         val eksternPeriodeMedStønadstypeSP = periodeService.hentPeriodeFraVedtakForSkolepenger(request.personIdent)
 
-        return OvergangsstønadOgSkolepengerResponse(request.personIdent, eksternPeriodeMedStønadstypeOS + eksternPeriodeMedStønadstypeSP)
+        return EksternePerioderMedStønadstypeResponse(request.personIdent, eksternPeriodeMedStønadstypeOS + eksternPeriodeMedStønadstypeSP)
     }
 
+    fun hentPerioderForYtelser(request: EksternePerioderForStønadstyperRequest): EksternePerioderMedStønadstypeResponse {
+        val periodeFraRequest = Datoperiode(fom = request.fomDato ?: LocalDate.MIN, tom = request.tomDato ?: LocalDate.MAX)
+        val stønadstyper =
+            if (request.stønadstyper.isEmpty()) {
+                listOf(StønadType.OVERGANGSSTØNAD, StønadType.BARNETILSYN, StønadType.SKOLEPENGER)
+            } else {
+                request.stønadstyper
+            }
+
+        val stønadPerioder =
```

Comment:
Antar det er diskutert med tilleggs-stønader - at vi gjør filtrering? Vi kunne jo alltid sende alt - de kunne filtert selv? :D

Reply:
Det er sant. Nå er det valgfritt, for hvis de ikke sender med noen filtrering blir alt sendt med. Tror kanskje det var litt ulik filtrering ut ifra hvilken visning de har. Ytelsesmessig kan det hende det er litt bedre å ikke måtte hente alt fra infotrygd, men mulig det er så lite forskjell at man ikke trenger å ta det med i vurderingen. Dette var noe som spesifikt var ønsket, men ikke noe jeg diskuterte spesifikt. Nå har jeg jo kodet det klart, så det er vel kanskje like greit å bare la det stå?